libs/sdk-ui-dashboard/src/presentation/filterBar/attributeFilter/CustomDropdownActions.tsx

```diff
@@ -14,9 +19,11 @@ import {
     selectIsKPIDashboardDependentFiltersEnabled,
     useDashboardSelector,
 } from "../../../model";
-import { useAttributes } from "../../../_staging/sharedHooks/useAttributes";
 
-function useIsConfigButtonVisible(filterDisplayFormRef: ObjRef) {
+function useIsConfigButtonVisible(
+    filterDisplayFormRef: ObjRef,
+    attributes: IAttributeMetadataObject[] | undefined,
```

Comment:
can be simplified by `attributes?: IAttributeMetadataObject []`

Reply:
Will need to still have `useAttributes` hook here, since its no re-rendering properly once the filter item is added for the first time.

Reply:
Updated.